slices/unzip.yaml

```
@@ -0,0 +1,26 @@
+package: unzip
+
+essential:
+  - unzip_copyright
+
+slices:
+  bins:
+    essential:
+      - libbz2-1.0_libs
+      - libc6_libs
+    contents:
+      /usr/bin/funzip:
+      /usr/bin/unzip:
+      /usr/bin/unzipsfx:
+      # Should be a hard link, but Chisel does not yet support hard links
```

Let's add a TODO comment so we can track this.

```suggestion
      # Should be a hard link, but Chisel does
      # not yet support hard links.
      # TODO update this to a hard link once a
      # newer version of Chisel is released with
      # https://github.com/canonical/chisel/commit/9d9d2144cf47586d02ff7566b8764e70057987a4
```